ci/filter.nix

```diff
@@ -113,6 +113,7 @@ let
     "ocaml-migrate-parsetree"
     "wasm"
     "pythonlib"
+    "camlp5"
 
     # Broken on OCaml 5.00
     "fontconfig"
```

Comment:
can you migrate these to the ocaml 5 extra ignores?

Reply:
Done